docs/client-side/yeoman.rst

```diff
@@ -0,0 +1,260 @@
+
+.. image:: yeoman/_static/yeoman-nodejs-setup.png
+ 
+2.	Open a command line such as Windows PowerShell, and run the command to list all NPM packages found globally. The list will also show you where the packages are located on your system.
+
+.. code-block:: console
+
+ npm ls -g
+  
+3.	Run the command to install the Gulp command line interpreter:
+
+.. code-block:: console
+
+ npm install -g gulp-cli
+
+4.	On the command line, install the ASP.NET generator:  
+
+.. code-block:: console
+
+ npm install -g generator-aspnet
+
+.. note:: The **–g** flag installs the generator globally so that you can use it from any path on your system. The **yo** command and supporting Node.js modules are installed automatically.
+ 
+5.	Make a new directory where your project will be generated:
+
+.. code-block:: console
+
+ mkdir c:\MyYo
+
+6.	On the command line, make the new directory the current directory.
+
+.. code-block:: console
+ 
+ cd c:\MyYo
+
+7.	Run the ``yo`` command and pass the name of the generator.
+
+.. code-block:: console 
+
+ yo aspnet
+
+8.	The aspnet generator displays a menu. Leave the default **Empty Application** type selected and press ``Enter``.
+
+9.	The generator prompts for the name of the application. Leave the default and press ``Enter``.
+
+.. image:: yeoman/_static/yeoman-yo-aspnet.png
+
+10.	Open Visual Studio 2015. From the File menu select :menuselection:`Open --> Project/Solution`.
+
+11.	In the Open Project dialog, navigate to the ``project.json`` file, select it and click the **Open** button. In the Solution Explorer, the project should look something like the screenshot below.
+
+ .. image:: yeoman/_static/yeoman-empty-application.png
+ 
+12.	This example doesn't use the cross-platform Kestrel server, so open ``project.json`` for editing and remove both references to **kestrel** from the **dependencies** and **commands** sections. Save the ``project.json`` file.
+
+13.	Press **F5** to run the project. The default Welcome screen should display.
+ 
+ .. image:: yeoman/_static/yeoman-welcome.png 
+ 
+Integrating Generated Projects with Visual Studio
+-------------------------------------------------
+
+Generator authors may not have Visual Studio in mind when they create a new Yeoman generator. Each generator implements its own distinct, opinionated, grouping of technology. As long as the generator creates a working set of files for the purpose, the generator is successful. For example, the `Yeoman team <http://yeoman.io/>`_ has a webapp generator that scaffolds a front-end web application. The generator includes a build process with Grunt or Gulp and Bower and a number of other features. The generated files have no concept of "project". The generated files (see screenshot below) don't contain a `project.json` and you won't find project files that are specific to Visual Studio.
```

Comment:
Clarify which generator is this specifically referring to. I assume generator-webapp?